nessie/src/test/java/org/apache/iceberg/nessie/TestNessieTable.java

```diff
@@ -407,7 +407,7 @@ public void testDropTable() throws IOException {
     table.newAppend().appendFile(file1).appendFile(file2).commit();
 
     // delete file2
-    table.newDelete().deleteFile(file2.path()).commit();
```

Comment:
Same question here.

Reply:
it's not a required change, but I wanted to move existing places to the non-path-based delete path